onnxscript/function_libs/torch_lib/ops/common.py

```diff
@@ -78,3 +80,35 @@ def constant(
         A constant node.
     """
     return op.Constant(value=ir.tensor(array, dtype=ir.DataType(dtype)))
+
+
+def merge_dims(dims: Sequence[int | INT64]) -> INT64:
```

Comment:
I wonder if this is worth it (except in the case when all dims are int constants) ... when I see a concat to produce the shape of a 5-dimensional tensor, I prefer seeing `Concat(dim0, dim1, dim2, dim3, dim4)` instead of `Concat(dim0, dim1, dim2, dim3and4)` ... it is immediately suggestive of a 5-dimensional shape and we can click on dim3 or dim4 as we want, etc.